insolar/payload/error_code.go

```diff
@@ -17,10 +17,26 @@
 package payload
 
 const (
-	CodeUnknown      = 0
-	CodeDeactivated  = 1
-	CodeFlowCanceled = 2
-	CodeNotFound     = 3
-	CodeNoPendings   = 4
-	CodeNoStartPulse = 5
+	CodeUnknown                = 1000
```

Comment:
whats's the point starting error constants from 1000 instead of 0?

Reply:
first two digits is prefix, last two is code, common practice